### PR TITLE
Add Contributing md file

### PR DESCRIPTION
**Pull request by amitkumar-y**
Added the contributing md file, let me know if you want any change.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    ```
+
+9. Push the changes to the remote repository
+
+    ```sh
+    $ git push -u origin <branch-name>
+    ```
+
+10. Submit a pull request to the upstream repository.
+11. Title the pull request with a short description of the changes you made and **request a review from a maintainer/reviewer**. For example, you can title an issue like so "Added John Doe to the project". In the description of the pull request, be sure to explain the changes that you made and any questions you have for the maintainer. It's OK if your pull request is not perfect (no pull request is), the reviewer will be able to help you fix any problems and improve it!
```
Comment (aaryan2134) [inline CONTRIBUTING.md]: Please change the example to something which is more appropriate to the repository.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+12. Wait for the pull request to be reviewed by a maintainer.
+13. Make changes to the pull request if the reviewing maintainer recommends them.
+14. Celebrate your success after your pull request is merged! 🎉
+
+## Reviewer/Maintainers
+
+  **GitHub**: [Aaryan Arora](https://github.com/aaryan2134) <br>
+  **Email**: aaryan2134@gmail.com <br>
+  **Quora**: https://www.quora.com/profile/Aaryan-Arora-3 <br>
```
Comment (aaryan2134) [inline CONTRIBUTING.md]: Please remove Quora from here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  **Linkedin**: https://www.linkedin.com/in/aaryan-arora-a956b8203/
+
+### Our Responsibilities
```
Comment (aaryan2134) [inline CONTRIBUTING.md]: Please remove this section

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+# What do I need to know to help?
+
+When contributing to this repository, please feel free to discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
```
Comment (aaryan2134) [inline CONTRIBUTING.md:4]: Please add a section on "What can you contribute?"
Which would include correcting any errors, missing information, solutions with explanations, links to online available solutions, etc

**Review by aaryan2134 — CHANGES_REQUESTED** [review]
Please make these changes for merging the request
Thank you for your effort

**Comment by amitkumar-y** [comment]
Hi @aaryan2134 ,
I have incorporated the suggestions that you had provided, take a look at the updated Changes and let me know if you need any more changes.

**Comment by aaryan2134** [comment]
Great work. Thank you for contributing.

**Pull request merged**